### PR TITLE
Add country profiles slug page

### PR DESCRIPTION
**Pull request by DonTouzi**

### CODE REVIEW

**Review by finnwoelm-sdsn — CHANGES_REQUESTED** [review]
This looks great, @DonTouzi!!

For `next.config.js`, can you export the profile pages just for countries? At the moment, you can visit region profiles & income profiles. But for now, we only want to have countries.

For example: https://deploy-preview-20--sdr-twenty-one.netlify.app/profiles/low-income-countries

File: next.config.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ module.exports = {
   ) => {
     const paths = defaultPathMap;
 
+    const { regions } = require("./public/data/regions.json");
```
[the remark this answers is not in the document]
Comment (DonTouzi) [reply next.config.js:25]: @FinnWoelm, you are right :), done

**Comment by finnwoelm-sdsn** [comment]
Hi @DonTouzi, thank you for the fix! Now, the Cypress test is failing — do you want to take a look and see if you can find the problem and solution?

(Let me know if you get stuck)

**Comment by DonTouzi** [comment]
> Hi @DonTouzi, thank you for the fix! Now, the Cypress test is failing — do you want to take a look and see if you can find the problem and solution?
> 
> (Let me know if you get stuck)

Yes, i'll check now

**Comment by DonTouzi** [comment]
Hi @FinnWoelm, it's fixed :), i think you can merge

**Review by finnwoelm-sdsn — APPROVED** [review]
Great job! Perfect! Merging now 👍

**Pull request merged**